Tools/TestWebKitAPI/cocoa/TestWKWebView.mm

```diff
@@ -1244,6 +1244,21 @@ - (void)focus
     return samples;
 }
 
+- (RetainPtr<_WKFrameTreeNode>)frameTree
+{
+    __block RetainPtr<_WKFrameTreeNode> result;
+    __block bool isDone = false;
+
+    [self _frames:^(_WKFrameTreeNode *tree) {
+        result = tree;
+        isDone = true;
+    }];
+    TestWebKitAPI::Util::run(&isDone);
+
```

Comment:
nit: extra newline.  Also, we don't need isDone to be separate from result.  We could just spinRunLoop until result is non-null.

Reply:
There's multiple examples I can find of a RetainPtr<> being the actual result and a separate `done` variable being required.
I tried removing the `done` and using the smart pointer directly instead and that wouldn't build.